src/622/modules/fios-sequenciados/index.test.js

```diff
@@ -0,0 +1,429 @@
+const seqwires = require('./index')
+
+describe('fios-sequenciados', () => {
+  describe('primeira chamada', () => {
+
+    const lastStep = {}
+    const currentStep0 = {
+        "1" : [null, null, null],
+        "2" : ["black", "C", null],
+        "3" : ["red", "C", null]
+    }
```

Comment:
Se as variáveis são usadas apenas em um teste, é melhor que fiquem dentro do escopo do `it`. Colocamos assim fora só quando elas são usadas em mais de um teste.